### PR TITLE
Update Sequential broken link.

### PR DESCRIPTION
**Pull request by shmishra99**
I have updated the broken link for sequential model from old link https://js.tensorflow.org/api/0.15.1/#class:Sequential to new link https://js.tensorflow.org/api/latest/#sequential
Please do the needful. Thank You !

### CODE REVIEW

File: docs/guide/models_and_layers.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ There are two ways to create a model using the Layers API: A _sequential_ model,
 
 ### The sequential model
 
-The most common type of model is the <code>[Sequential](https://js.tensorflow.org/api/0.15.1/#class:Sequential)</code> model, which is a linear stack of layers. You can create a <code>Sequential</code> model by passing a list of layers to the <code>[sequential()](https://js.tensorflow.org/api/0.15.1/#sequential)</code> function:
+The most common type of model is the <code>[Sequential](https://js.tensorflow.org/api/latest/#sequential)</code> model, which is a linear stack of layers. You can create a <code>Sequential</code> model by passing a list of layers to the <code>[sequential()](https://js.tensorflow.org/api/0.15.1/#sequential)</code> function:
```
Comment (pyu10055) [inline docs/guide/models_and_layers.md:18]: can you update the second link as well? thanks

Comment (shmishra99) [reply docs/guide/models_and_layers.md]: Hi @pyu10055 ,
I am very sorry that I didn't check the other broken sequential link. I have updated the link now. Please do the needful. Thank You!

**Review by fengwuyao — APPROVED** [review]

**Review by mattsoulanille — APPROVED** [review]
LGTM

**Pull request merged**